### ISSUE TITLE
Keyboard type rate is very high

### ISSUE THREAD

**Issue by asinha94** [asinha94/asos#13]
When I type in QEMU or Bochs it seems to be repeating keys indicating that the repeat rate is quite high or the ignore ms is too low. This should be looked into once a userspace is up and running

**Comment by asinha94** [comment]
It could be related, but on this line [here](https://github.com/asinha94/asos/blob/0cefe71c5870828db451679757150b541f39b166/kernel/drivers/keyboard/keyboard.c#L41) for some reason we explicitly set the bit which tells us if a multi-char code was released, to high, which I don't remember doing. Some investigation would be nice

**Comment by asinha94** [comment]
Nvm I realized I did this because of the location of the scan-code. In my scan-code table I wanted to avoid as much duplication as possible. So a pressed/released scan-code will get the same entry from the table. The problem with the multi-char codes is that the second-digit overlaps with some of the single-char scan-codes, so I purposefully use the released scan-code variant which has the released bit set. I should explain this in the code
